apps/table.html

```diff
@@ -122,9 +144,9 @@ <h1 class="h1">caMicroscope</h1>
 		<button type="button" class="btn btn-success float-right mb-2"
 			onclick="(()=>{location.reload();return false;})()"> <i class="fas fa-sync-alt"></i> Reload</button>
 		<button type="button" class="btn btn-primary float-right mb-2 mr-2" data-toggle="modal"
-			data-target="#upload-dialog"> <i class="fas fa-upload"></i> Upload</button>
+			data-target="#upload-dialog" onclick="hidePostButton(); hideCheckButton(); resetUploadForm();"" <i class="fas fa-upload"></i> Upload</button>
```

Comment:
@cjchirag7 
You missed a closing tag here (>), and also a extra quote is present( " ) due to which the upload icon is no longer visible.

Reply:
on it.